source/features/show-open-prs-of-forks.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {isRepoWithAccess} from '../libs/page-detect';
+import {getUsername} from '../libs/utils';
+
+function findForkedRepo(): string | undefined {
+	const forkSourceElement = select<HTMLAnchorElement>('.fork-flag a');
+	if (forkSourceElement) {
+		return forkSourceElement.pathname.slice(1);
+	}
+
+	return undefined;
+}
+
+interface SearchResponse {
+	issueCount: number;
+	edges: Array<{
+		node: {
+			url: string;
+		};
+	}>;
+}
+
+function getUserPullRequestsURL(forkedRepo: string, user: string): string {
+	return `https://github.com/${forkedRepo}/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+author%3A${user}`;
```

Comment:
Do I need to consider anything around this URL regarding GitHub Enterprise support? Would `location.origin` be more appropriate here than hardcoding the github.com domain?

Reply:
Yes